src/io.rs

```diff
@@ -106,3 +109,78 @@ impl<D: Drive> Future for Print<D> {
         }
     }
 }
+
+/// A handle to the standard output of the current process.
+pub struct Stdout<D: Drive> {
+    ring: Ring<D>,
+    buf: Buffer,
+}
+
+/// Constructs a new handle to the standard output of the current process using the demo driver.
+/// ```no_run
+/// use ringbahn::io;
+///
+/// # use futures::AsyncWriteExt;
+/// # fn main() -> std::io::Result<()> { futures::executor::block_on(async {
+/// io::stdout().write(b"hello, world").await?;
+/// # Ok(())
+/// # })
+/// # }
+/// ```
+// TODO synchronization note?
+pub fn stdout() -> Stdout<DemoDriver> {
+    Stdout::run_on_driver(DemoDriver::default())
+}
+
+impl<D: Drive> Stdout<D> {
+    pub fn run_on_driver(driver: D) -> Stdout<D> {
```

Comment:
I think I would like to see this more parallel to the stdout function; that is, have a public free function called `stdout_on_driver` instead of having a free function and a constructor.

Reply:
You got it, done in b4b91e8